site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -97,12 +113,14 @@ chrome.action.onClicked.addListener(async () => {
 });
 ```
 
-For complete examples, see the [offscreen-clipboard][gh-offscreen-clipboard] and [offscreen-dom][gh-offscreen-dom] demos on GitHub.
+For complete examples, see the [offscreen-clipboard][gh-offscreen-clipboard] and
+[offscreen-dom][gh-offscreen-dom] demos on GitHub.
 
 ### Before Chrome 116: check if an offscreen document is open
 
 [`runtime.getContexts()`][runtime-get-contexts] was added in Chrome 116. In earlier versions of
-Chrome, you can check for the existence of the offscreen document using [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll):
+Chrome, use [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll)
+to check for an existing offscreen document:
```

Comment:
```suggestion
to check for an existing offscreen document. To cover both Chrome 116 or later, and earlier than 116, you can do something like the code below. Alternatively, you can set the [`"minimum_chrome_version"`](/docs/extensions/mv3/manifest/minimum_chrome_version/) to 116.
```

Reply:
Does this change still apply to the new code?

Reply:
Yes. Ask me about feature detection dome time.

Reply:
Even though Chrome autoupdates, enough users have it blocked that older versions need to be taken into account in web and extension development. An enterprise extension can set a minimum. The developer of a web store extension can set a minimum version but probably shouldn't.